docs/BIDS.md

```diff
@@ -40,3 +44,22 @@ BIDS_Dataset/
 ├── ...
 └── participants.tsv
 ```
+
+## Validation of BIDS datasets
+
+[bids-validator](https://github.com/bids-standard/bids-validator) can be run to ensure that a dataset is BIDS-compliant.
+
+Some errors and files can be ignored by the validator in respectively `.bids-validator-config.json` and 
+`.bidsignore` files at the root of the BIDS folder.
+These files are automatically generated by clinica converters to ignore the following issues:
+- Won't fix errors:
+    - PET modality is not BIDS-compliant yet, it will be after merging [BEP 009](https://github.com/bids-standard/bids-specification/pull/633)
+    (`pet/` in `.bidsignore` / `SCANS_FILENAME_NOT_MATCH_DATASET` in `.bids-validator-config.json`).
```

Comment:
I would change the wording about it:

> PET modality is not officially supported by BIDS but is currently under revisions. See [BEP 009](https://github.com/bids-standard/bids-specification/pull/633) for details (`pet/` in `.bidsignore` / `SCANS_FILENAME_NOT_MATCH_DATASET` in `.bids-validator-config.json`). 

I am not sure that providing the JSON files for PET modality according to upcoming specifications of BIDS-PET will be straightforward to implement 🤔

Reply:
Done in https://github.com/aramis-lab/clinica/pull/144/commits/1e8870687de35755191595a3e75478a0be86a1a3